apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/matcher/FastPathMatcher.java

```diff
@@ -124,7 +135,7 @@ private boolean wildcardMatch(String pat, int p, String str, int s) {
     private boolean multiWildcardMatch(String pat, int p, String str, int s) {
         // End of pattern, just check the end of string is '/' quickly.
         if (p >= pat.length() && s < str.length()) {
-            return str.charAt(str.length() - 1) != '/';
+            return true;
```

Comment:
I think we can remove this check after we remove the last '/' both of pattern and value,

Reply:
ok